src/ctap2/commands/mod.rs

```diff
@@ -114,120 +112,17 @@ pub(crate) enum PinUvAuthResult {
     SuccessGetPinUvAuthTokenUsingPinWithPermissions,
 }
 
-pub(crate) trait PinUvAuthCommand: RequestCtap2 {
+/// Helper-trait for MakeCredential and GetAssertion, which share a common
+/// algorithm to determine pin_uv_auth_param from PIN or UV.
+pub(crate) trait McGaPinUvAuthCommand: RequestCtap2 {
```

Comment:
Without additional refactoring, it probably will still be restricted to `McGa`, because of `client_data_hash()` et al., but it still makes sense to separate `PinUvAuthParam` and `*Token` in the way you suggested. 

We can take a look at that trait again, once we add other commands. I'm still reverting the rename.